libr/bin/meson.build

```diff
@@ -131,6 +131,7 @@ r_bin = library('r_bin', files,
     r_magic,
     libr_shlr
   ],
+  dependencies: lmagics,
```

Comment:
Why no `[]`?

Reply:
I'm no expert of `meson`, but `lmagics` is already an array, so I didn't put `[]`.